src/Assets/greenFaded.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="375.041" height="185.43" viewBox="0 0 375.041 185.43">
+  <path id="Path_38" data-name="Path 38" d="M679.248,1235.562s86.553,3.181,149.016,31.724S910.078,1317.134,941.4,1346s36.658,43.84,67.955,47.866,44.936-17.586,44.936-17.586v44.714H679.248Z" transform="translate(-679.248 -1235.562)" fill="rgba(54,159,173,0.43)"/>
+</svg>
```

Comment:
There are some not used properties in your SVGs. These properties are created by design tools like Adobe XD.
For example, you can remove `id` and `data-name`.
Take a look at https://www.svgminify.com/